makefile

```diff
@@ -16,7 +16,7 @@ WORKLOAD =SSBM
 # add -EDTERABYTE if orderkey will execeed 32 bits (SF >= 300)
 # and make the appropriate change in gen_schema() of runit.sh
 # HYRISE: Change -O to -O3. Add -Wno-dangling-else, -Wno-int-to-void-pointer-cast, and -Wno-void-pointer-to-int-cast because they are all over the codebase.
-CFLAGS	= -O3 -DDBNAME=\"dss\" -D$(MACHINE) -D$(DATABASE) -D$(WORKLOAD) -Wno-dangling-else -Wno-int-to-void-pointer-cast -Wno-void-pointer-to-int-cast
+CFLAGS	= -O3 -DDBNAME=\"dss\" -D$(MACHINE) -D$(DATABASE) -D$(WORKLOAD) -Wno-dangling-else -Wno-int-to-void-pointer-cast -Wno-void-pointer-to-int-cast -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast
```

Comment:
Actually, why is `-O3` not also an optional flag just like the compiler? We would keep everything as is but still allow "better" builds for Hyrise.

(not that is matters ... won't block this PR).

Reply:
Thought about the same. However, we only call the SSB and JCC-H dgbens via CLI. No way we ever step into the code while debugging, so I see no advantage of not always using `-O3`.

Reply:
Do you think they want to debug this? I mean, we can add a switch for either settinf `-g` or `-O3`, but I guess anyone who wants to step into this ~abomination~ beauty can set compiler flags on their own.

Reply:
Hmm ... okay. When somebody really wants to debug, then she will need to dig into this part anyways. Good point.